geode-management/src/main/java/org/apache/geode/management/configuration/Pdx.java

```diff
@@ -31,8 +32,8 @@
 
   private Boolean readSerialized;
   private ClassName pdxSerializer;
+  private AutoSerializer autoSerializer;
   private Boolean ignoreUnreadFields;
-  private Boolean persistent;
```

Comment:
getting rid of "persistent" is good, but makes it harder for user to use "DEFAULT"  diskstore. Let's add a method called `setPersistent()`, which set the diskstore name to "DEFAULT"

Reply:
In this particular case, we should not make it easy to use the default disk store. The PCC team, for example, recently went through a difficult upgrade to switch pdx from the default disk store to a dedicated pdx disk store. Having the pdx types stores in the same disk store as the region data can cause issues down the road. So this particular change I think is good because it forces a user to explicitly say "I want my pdx to use the DEFAULT disk store".

If we decide to make it easy to use the default disk store for pdx then I'm not sure I like doing it by adding a setPersistent() method. This would only be of help to users of the java management api; not the rest management api users. I think we want to keep a close correspondence between the java and rest management apis. 

I also don't like us having methods that look like a settor that have no corresponding attribute. Users expect if they call setPersistent and then print out the config to see something like "persistent=true" not "diskStoreName=DEFAULT".

Reply:
after discussion with Gang, we agreed that it would be a good idea to forcing user to specify a diskstore name since we don't want them to use "DEFAULT" without justifying it.